### PR TITLE
Add support regexp for CORS settings

### PR DESCRIPTION
**Pull request by startpointforl**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,9 +24,21 @@ dapr init -s
 ### Launch dev
 
 docker-compose required
+docker start required (example `colima start`)
```
Comment (Jhoysbou) [inline CONTRIBUTING.md:27]: Not everyone uses colima

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,9 +24,21 @@ dapr init -s
...
 
 ```bash
 yarn build
-yarn setup:dev
 yarn dev
 ```
+
+After application start:
```
Comment (Jhoysbou) [inline CONTRIBUTING.md:34]:
```suggestion
After application starts for the first time
```

File: packages/api/src/main.ts [hunk 3]
```diff
@@ -5,8 +5,16 @@ import { config } from './config';
 
 async function bootstrap() {
     const app = await NestFactory.create(AppModule);
+
+    const origin = config.cors.origin
+        ? config.cors.origin.map((str) => {
+              const strEscaped = str.replace(/\./g, '\\.');
```
Comment (Jhoysbou) [inline packages/api/src/main.ts:11]: Why not `encodeURIComponent()`?
I think its more general

**Review by Jhoysbou — CHANGES_REQUESTED** [review]

File: packages/api/src/main.ts (hunk 3 above)
Comment (startpointforl) [reply packages/api/src/main.ts:11]: `encodeURIComponent()` don't escape `.`, but it's required for RegExp
image

**Review by Jhoysbou — APPROVED** [review]
👍

**Pull request merged**